### PR TITLE
feat(appeals/web): content changes for document process

### PR DESCRIPTION
**Pull request by AlexMoulin**
## Issue ticket number and link
https://pins-ds.atlassian.net/browse/BOAT-984
https://pins-ds.atlassian.net/browse/BOAT-986
https://pins-ds.atlassian.net/browse/BOAT-987

## Type of change 🧩

- [x] Bug fix (non-breaking change which fixes an issue)
- [ ] New feature (non-breaking change which adds functionality)
- [ ] Breaking change (fix or feature that would cause existing functionality to not work as expected)
- [ ] Documentation update
- [ ] Other (please explain in the description section above)

## Checklist before requesting a review

- [ ] I have performed a self-review of my own code
- [ ] I have double checked this work does not include any hardcoded secrets or passwords
- [ ] I have made corresponding changes to the documentation
- [ ] I have provided details on how I have tested my code
- [ ] I have referenced the ticket number above
- [ ] I have provided a description of the ticket
- [ ] I have included unit tests to cover any testable code changes

### CODE REVIEW

File: appeals/web/src/server/appeals/appeal-details/lpa-questionnaire/__tests__/lpa-questionnaire.test.js [hunk 1]
```diff
@@ -55,7 +55,7 @@ describe('LPA Questionnaire review', () => {
 			const element = parseHtml(response.text);
 
 			expect(element.innerHTML).toMatchSnapshot();
-		});
+		}, 10000);
```
Comment (james-hill-kc) [inline appeals/web/src/server/appeals/appeal-details/lpa-questionnaire/__tests__/lpa-questionnaire.test.js:58]: What's the reason for overriding the default timeout here? Is this a long running test?

**Review by james-hill-kc — APPROVED** [review]
Looks good to me! Just one question re. timeout in a test, but nothing that should block this being merged

File: appeals/web/src/server/appeals/appeal-details/lpa-questionnaire/__tests__/lpa-questionnaire.test.js (hunk 1 above)
Comment (AlexMoulin) [reply appeals/web/src/server/appeals/appeal-details/lpa-questionnaire/__tests__/lpa-questionnaire.test.js:58]: yes the test kept over running

Comment (AlexMoulin) [reply appeals/web/src/server/appeals/appeal-details/lpa-questionnaire/__tests__/lpa-questionnaire.test.js:58]: not sure why

**Pull request merged**